core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java

```diff
@@ -343,6 +343,8 @@ public Response getWebUIBlockInfo(@QueryParam("path") String requestPath,
                 uiFileInfo
                     .addBlock(blockMeta.getBlockLocation().tierAlias(), blockId, blockSize, -1);
               }
+            }
+            if(uiFileInfo.getBlockIds().isEmpty()) {
```

Comment:
spacing

Reply:
oh，I am very sorry. when i commit fix forgot add the 
**private final List<Long> mBlockIds = new ArrayList<>();**
to branch.
i will pay attention to it and add a unit test next time.